Controller/TwigExceptionController.php

```diff
@@ -28,31 +27,13 @@ class TwigExceptionController extends TemplatingExceptionController
     protected function createView($format, FlattenException $exception, $code, $parameters, Request $request, $showException)
     {
         $view = parent::createView($format, $exception, $code, $parameters, $request, $showException);
-
-        if ($this->getViewHandler()->isFormatTemplating($format)) {
-            $view->setTemplate($this->findTemplate($request, $format, $code, $showException));
-            $view->setData($parameters);
-        }
+        $view->setTemplate($this->findTemplate($request, $format, $code, $showException));
```

Comment:
We would then need to access the templating service during compilation. I don't think that this is a good idea.

A better approach would be do this when warming up the cache for a reasonable matrix of formats and status codes and then rely on the cached data if present.